src/System.Globalization.Extensions/tests/IdnMapping/IdnMappingIdnaConformanceTests.cs

```diff
@@ -41,25 +49,27 @@ public void GetAscii_Success()
         /// Same applies to Windows 10 >= 10.0.15063 in the IdnaTest_9.txt file
         /// </summary>
         [Fact]
-        public void GetUnicode_Succes()
+        public void GetUnicode_Success()
         {
-            foreach (IConformanceIdnaTest entry in Factory.GetDataset())
+            Assert.All(Factory.GetDataset().Where(e => e.UnicodeResult.Success && e.UnicodeResult.ValidDomainName), entry =>
             {
-                if (entry.UnicodeResult.Success && entry.UnicodeResult.ValidDomainName)
+                try
                 {
-                    try
-                    {
-                        var map = new IdnMapping { UseStd3AsciiRules = true, AllowUnassigned = true };
-                        var unicodeResult = map.GetUnicode(entry.Source);
+                    var map = new IdnMapping { UseStd3AsciiRules = true, AllowUnassigned = true };
+                    var unicodeResult = map.GetUnicode(entry.Source);
 
-                        Assert.Equal(entry.UnicodeResult.Value, unicodeResult, StringComparer.OrdinalIgnoreCase);
-                    }
-                    catch (ArgumentException)
+                    Assert.Equal(entry.UnicodeResult.Value, unicodeResult, StringComparer.OrdinalIgnoreCase);
+                }
+                catch (ArgumentException)
+                {
+                    if (!string.Equals(entry.UnicodeResult.Value, entry.Source, StringComparison.OrdinalIgnoreCase))
```

Comment:
If they are equal we just ignore the error?

Reply:
I'll look in master. Here I will port as is.